tooling/minimize/tests/pass/assert.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    // An assert is put before divisions, so this should panic but not create UB.
+    let _ = 42 / blackbox(0);
+}
+
+fn blackbox<T>(i: T) -> T {
+    i
+}
```

Comment:
The blackbox stuff shouldn't be needed.

Reply:
Added `#[allow(unconditional_panic)]` instead.